### PR TITLE
edits to the event stream documentation

### PR DESCRIPTION
**Pull request by camicano**

### CODE REVIEW

**Comment by camicano** [comment]
@Deborah-Digges please review 😄

File: articles/logs/streams/datadog.md [hunk 1]
```diff
@@ -59,6 +59,24 @@ To send Auth0 events to Datadog, you will need:
 
 Auth0 events are delivered to your server via a streaming mechanism that sends each event as it is triggered. If your server is unable to receive the event, Auth0 will try to redeliver it up to three times. If still unsuccessful, Auth0 will log the failure to deliver, and you will be able see these failures in the Health tab for your log stream.
 
+## Enhancement to log data
+
+Some of the unique value of Datadog as a monitoring tool, and specifically when it comes to their integrations, is the data enhancement they do to make sure customers can rely on specific data fields regardless of the system they are integrating with. As part of this Log Streaming integration, Datadog has added some enhancements to our data. Here is the list of the new fields found in our logs when using the Log Streaming integration with Datadog:
+
+| Fiels | Auth0 attribute |
```
Comment (Deborah-Digges) [inline articles/logs/streams/datadog.md]: ``
```suggestion
| Fields | Auth0 attribute |
```

Comment (Deborah-Digges) [reply articles/logs/streams/datadog.md]: Could we use column names like `Datadog log attribute`, `Auth0 log attribute` to be clearer?

File: articles/logs/streams/datadog.md [hunk 2]
```diff
@@ -59,6 +59,24 @@ To send Auth0 events to Datadog, you will need:
...
+| Official Log date | data.date |
+| data.network.ip | data.ip |
+| data.client.geoip |	data.ip (parsed) |
+| http.useragent |	data.user_agent |
+| http.useragent_details |	data.user_agent (parsed) |
+| usr.id |	data.user_name |
+| usr.name |	data.user_name |
+| usr.email |	data.details.request.auth.user.email (when available) |
+| evt.name | Event name expanded (list can be found [here](/logs/references/log-event-type-codes)) |
+| message |	Event description  (list can be found [here](/logs/references/log-event-type-codes))|
```
Comment (Deborah-Digges) [inline articles/logs/streams/datadog.md]: We could also link them to where the transformations are present if they'd like to review it:

For US: https://app.datadoghq.com/logs/pipelines
For EU: https://app.datadoghq.EU/logs/pipelines

File: articles/logs/streams/datadog.md [hunk 3]
```diff
@@ -59,6 +59,24 @@ To send Auth0 events to Datadog, you will need:
...
 Auth0 events are delivered to your server via a streaming mechanism that sends each event as it is triggered. If your server is unable to receive the event, Auth0 will try to redeliver it up to three times. If still unsuccessful, Auth0 will log the failure to deliver, and you will be able see these failures in the Health tab for your log stream.
 
+## Enhancement to log data
+
+Some of the unique value of Datadog as a monitoring tool, and specifically when it comes to their integrations, is the data enhancement they do to make sure customers can rely on specific data fields regardless of the system they are integrating with. As part of this Log Streaming integration, Datadog has added some enhancements to our data. Here is the list of the new fields found in our logs when using the Log Streaming integration with Datadog:
+
+| Fiels | Auth0 attribute |
+|---------|-------------|
+| Official Log date | data.date |
+| data.network.ip | data.ip |
```
Comment (Deborah-Digges) [inline articles/logs/streams/datadog.md]:
```suggestion
| network.client.ip | data.ip |
```

This is the correct Datadog field according to the configuration I just checked?

File: articles/logs/streams/datadog.md [hunk 4]
```diff
@@ -59,6 +59,24 @@ To send Auth0 events to Datadog, you will need:
...
+| data.client.geoip |	data.ip (parsed) |
```
Comment (Deborah-Digges) [inline articles/logs/streams/datadog.md]:
```suggestion
| network.client.geoip |	data.ip (parsed) |
```

File: articles/logs/streams/datadog.md [hunk 5]
```diff
@@ -59,6 +59,24 @@ To send Auth0 events to Datadog, you will need:
...
+| http.useragent |	data.user_agent |
+| http.useragent_details |	data.user_agent (parsed) |
+| usr.id |	data.user_name |
+| usr.name |	data.user_name |
+| usr.email |	data.details.request.auth.user.email (when available) |
+| evt.name | Event name expanded (list can be found [here](/logs/references/log-event-type-codes)) |
```
Comment (Deborah-Digges) [inline articles/logs/streams/datadog.md]: The updated mapping is `data.type to evt.name`

File: articles/logs/streams/datadog.md (hunk 2 above)
Comment (camicano) [reply articles/logs/streams/datadog.md]: good idea

**Review by Deborah-Digges — DISMISSED** [review]

**Comment by camicano** [comment]
@erjohnson can you merge this please?

**Review by rkhoriander — APPROVED** [review]

**Comment by rkhoriander** [comment]
Edits also made in Contentful

**Pull request merged**